### ISSUE TITLE
Warum nicht im Installer?

### ISSUE THREAD

**Issue by ghost** [yakamara/ytraduko#14]
ist doch ein praktisches Tool ;-)

**Comment by staabm** [comment]
Wir verwenden das tool um redaxo und seine addons selbst zu übersetzen.

Hast du fuer dich sinnvolle use-cases um es für dich zu verwenden?

**Comment by ghost** [comment]
Ja, bei der eigenen Addonentwicklung ist es sehr nützlich. Es entfällt ja die Erstellung der lang-files. Zudem ist es schön übersichtlich.

**Comment by gharlan** [comment]
Also wir haben hier eine zentrale Seite dafür: http://traduko.redaxo.org/index.php?page=ytraduko
Da bekommen die Übersetzer Logins und können dann alle Addons übersetzen. Und es gibt dann noch im Hintergrund einen Bot, der Pull Requests in den jeweiligen Addon-Repos öffnen kann.

Aber wenn Bedarf daran besteht, das Addon auch selbst zu nutzen, können wir es auch gerne im Installer veröffentlichen!

**Comment by ghost** [comment]
Ich habe es hier gefunden: https://github.com/FriendsOfREDAXO/tricks/blob/master/developer_resources.md 
Sehr hilfreich. Ich wundere mich nur, warum das nicht auf der offiziellen Seite ist.
